source/change-streams/change-streams.rst

```diff
@@ -129,6 +129,9 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
 
     /**
      * Describes the type of operation represented in this change notification.
+     *
+     * @note: Whether a change is reported as an event of the operation type
+     *       `update` or `replace` is a server implementation detail.
```

Comment:
The indentation is intentional. It is consistent with the indentation used in similar situations in the Java driver code. Please tell me if this style is surprising or unwelcome, and I will remove the indentation.

Reply:
I'm not sure why the Java driver docs do this, but unless there's precedent for doing so in this spec, I'd prefer to remove the additional spaces.

Reply:
I understand why this was originally indented and I think it has more to do with `@note` being an annotation instead of just prose text within the doc block. If we look at [doc blocks in the CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read), you'll see that most notes are written as a regular paragraph starting with "Note:". Some `@note` annotations were added to the spec in https://github.com/mongodb/specifications/commit/bb8677b9abb3440ca7e843d5f4ca554b8844e905 but they were one-liners.

IMO, it'd be preferable to just change this to "Note:" and leave it as a prose paragraph.

Reply:
I removed indentation.

@jmikola 
Extracting "Note:" out of `/** ... */` scatters the specification of `ChangeStreamDocument.operationType` to multiple places. This, I believe, hurts readability by improving the chances to miss a piece of information about `operationType` when looking through the document.